Controller/Dashboard.php

```diff
@@ -0,0 +1,51 @@
+<?php declare(strict_types=1);
+
+namespace KaroIO\MessengerMonitorBundle\Controller;
+
+use KaroIO\MessengerMonitorBundle\FailedMessage\FailedMessageRepository;
+use KaroIO\MessengerMonitorBundle\Locator\ReceiverLocator;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Messenger\Transport\Receiver\MessageCountAwareInterface;
+use Twig\Environment;
+
+class Dashboard
+{
+    private $twig;
+    private $receiverLocator;
+    private $failedMessageRepository;
+
+    public function __construct(
+        Environment $twig,
```

Comment:
The bundle now depends on `symfony/twig-bundle` - it'll need to be in the composer.json file

Reply:
And unrelated to your PR, the composer.json file needs a `php` requirement - probably `"php": "^7.1.3",` to match Symfony 4.4

Reply:
should i search for `use Symfony\Component` in the project and add all used components in the `composer.json`?

Reply:
up!

Reply:
Yea... that would be the *best* thing to do. I usually allow my tests to fail and notify me of any missing dependencies... but it's more responsible to do what you're saying here.